app/command_utils.cc.template

```diff
@@ -29,6 +29,7 @@ COMMAND_IDS
 
 }  // namespace
 
+// TODO(sko) We should return reference so that we can avoid copying all data.
```

Comment:
I think (but I could be wrong) that only a new `base::span` is created here, which is just a `pointer` + `size`, so we don't actually copy all the data here.

Reply:
Oh, yeah. You're totally right! 👍